packages/code-du-travail-frontend/src/common/RelatedItems.js

```diff
@@ -71,7 +72,7 @@ export const RelatedItems = ({ items = [] }) => {
 
                   return (
                     <StyledLinkItem key={slug || url}>
-                      <Link href={href} passHref>
+                      <Link href={href} as={selection} passHref>
```

Comment:
I think you might be missing the first "/" in the selection variable, but again, it might work without the "as" if you put the "#" directly in the href